### PR TITLE
Integrate live-common library

### PR DESCRIPTION
**Pull request by gre**
- i wasn't able to import testnet yet, it hangs forever for me. so i'm not sure if there is not broken changes i did here & we need to test everything before merging.
- it likely won't be compatible with existing data because data model has changed (coinType is gone & the AccountRaw needs to have a currencyId, i have not impleemnted a "migration" since we are still in development)

### CODE REVIEW

**Comment by meriadec** [comment]
Also, we seems to have regression on our [SelectCurrency](https://ledgerhq.github.io/ledger-live-desktop/?selectedKind=Components&selectedStory=SelectCurrency&full=0) component. Not sure if related.

File: package.json [hunk 1]
```diff
@@ -12,7 +12,6 @@
     "dist": "yarn compile && electron-builder",
     "test": "jest",
     "flow": "flow",
-    "flow-typed": "flow-typed install -s --overwrite",
     "lint": "eslint src webpack .storybook",
     "postinstall": "bash ./scripts/postinstall.sh",
     "precommit": "lint-staged",
```
[the remark this answers is not in the document]
Comment (gre) [reply package.json]: yes I agree, we already have PR / github flow to do the validation work

File: src/actions/counterValues.js [hunk 2]
```diff
@@ -20,18 +21,23 @@ export const updateCounterValues: UpdateCounterValues = payload => ({
...
+}
+
+export type FetchCounterValues = (
+  currencies?: Currency[],
+) => (Dispatch<*>, () => State) => Promise<any>
+
+export const fetchCounterValues: FetchCounterValues = currencies => async (dispatch, getState) => {
+  const state = getState()
+  const currency = counterValueCurrencySelector(state)
+  if (!currency) return
```
[the remark this answers is not in the document]
Comment (gre) [reply src/actions/counterValues.js]: if `if (..) return` is usually the only one i accept myself to not use `{}` but I can definitely

**Comment by gre** [comment]
@meriadec ok i'll look on monday about this regression.

**Comment by gre** [comment]
gre#9 🤣

**Comment by gre** [comment]
meriadec#1

**Pull request merged**